boxsdk/util/translator.py

```diff
@@ -183,7 +166,14 @@ def translate(self, session, response_object):
             return response_object
 
         translated_obj = {}
+        object_type = response_object.get('type', '')
+        if object_type is not None:
+            # Parent classes have the ability to "blacklist" fields that they do not want translated
+            blacklisted_fields = self.get(object_type)._untranslated_fields or []  # pylint: disable=protected-access
```

Comment:
Perhaps we could have a property for reading this so we don't have to access the private field here.